nbuild/stdenv/package.py

```diff
@@ -86,9 +86,9 @@ def do_package(builder):
             gnuflags = '-O2 -s -m64 -mtune=generic'
 
             # Pre-processor
-            os.environ['CPP'] = f'{os.environ["TARGET"]}-cpp'
-            os.environ['HOSTCPP'] = f'{os.environ["HOST"]}-cpp'
-            os.environ['CPPFLAGS'] = gnuflags
+            # os.environ['CPP'] = f'{os.environ["TARGET"]}-cpp'
```

Comment:
Why did you commented all this?

Reply:
Because on my system this wouldn't work, as I don't have those binaries. I haven't found any other solution yet.
(Well I could just make symlinks, but I'm not sure if I'm at fault or not for not having them)

Reply:
It is important those items are prefixed accordingly to their target architecture in order to avoid ambiguities while cross compiling. Without them, you'll end up using your host amd64 gcc to compile, for example, arm binaries.

The quick & dirty solution would be to add symlinks in a separate folder, in your home for example, and add this folder to your $PATH (as you said).